lxd/db/containers.go

```diff
@@ -169,8 +169,8 @@ type InstanceBackupArgs struct {
 	CompressionAlgorithm string
 }
```

Comment:
@freeekanayaka can this file be renamed to instances.go?

Reply:
Yes, I was going to perform file renames separately tho, if that's ok.